.github/workflows/compute.yml

```diff
@@ -0,0 +1,46 @@
+name: "the name of the plugin"
+
+on:
+  workflow_dispatch:
+    inputs:
+      stateId:
+        description: "State Id"
+      eventName:
+        description: "Event Name"
+      eventPayload:
+        description: "Event Payload"
+      settings:
+        description: "Settings"
+      authToken:
+        description: "Auth Token"
+      ref:
+        description: "Ref"
+
+jobs:
+  compute:
+    name: "plugin name"
+    runs-on: ubuntu-latest
+    permissions: write-all
+    env:
```

Comment:
To be removed

Reply:
I don't understand, you mean remove `name, permissions and env`? It needs `runs-on` or am I totally misunderstanding your comment

Reply:
Apologies, was talking about the `UBIQUIBOT_TOKEN` commented one line too high.

Reply:
ah no probs. And that's only because it's injected into the env further down right? Or should that reference be removed too because it's being passed via the payload?

```ts
 - name: execute plugin
        run: npx tsx ./src/main.ts
        id: plugin-name
        env:
          UBIQUIBOT_TOKEN: ${{ secrets.UBIQUIBOT_TOKEN }}
```

Reply:
Because for now at least, we should only be using the token contained within the payload, `authToken`.